mantid-development/readme.md

```diff
@@ -37,25 +37,37 @@ mounted to locations on the host filesystem. Reasons being:
 - Running a container built or packaged Mantid on the host (assuming an
   appropriate host system and base image)
 
-Of course Docker volumes could also be used for the build and external data
-directories if you will only ever do container builds.
+To ensure file permissions are handled correctly if mapping volumes to your host
+filesystem you must pass the `PUID` and `PGID` environment variables when
+starting a container, these should be set to your user ID and group ID
+respectively.
 
 The container can be run like this (replacing `centos7` with the base OS
 you care about):
 
 ```sh
-docker run --rm -it \
-  -v /path/to/mantid/source:/mantid_src \
-  -v /path/to/mantid/build:/mantid_build \
-  -v /path/to/mantid/data:/mantid_data \
+docker run \
```

Comment:
Should we suggest people put in an alias for this? It's getting quite cumbersome to type although most people would probably just go through history or autocomplete I guess.

Reply:
Yep, it would be better to mention this. That's what I've been doing and just assumed that would be done (hence the long parameter names and multi line formatting).

Reply:
:man_facepalming: I even have scripts in the repo for this (`run.sh`) No idea why I never mentioned them...

Reply:
No worries :)